### ISSUE TITLE
Grouped form controls missing an accessible name - Siteimprove - group role on forms

### ISSUE THREAD

**Issue by GaryRidgway** [uiowa/uiowa#9293]
Brandon E came across an issue Siteimprove is flagging on the Housing website. It’s on this page https://housing.uiowa.edu/residence-halls and concerns the form on the left side.

Siteimprove Error title: Grouped form controls missing an accessible name
Grouped form controls can include elements such as checkboxes or radio buttons where the user needs to make a selection. Each group should have an accessible name so that screen reader users can understand the group’s purpose.

Why is this an issue? When grouped form controls have an accessible name, it helps a screen reader user to make sense of a form. The accessible name is read out to the user and describes the purpose of related controls such as a set of options, credit card details, or an address.
If the form is long or complicated, it also helps to distinguish one group of form controls from another.

Help Center: [Grouped form controls missing an accessible name, explained]

<img width="1306" height="1192" alt="Image" src="https://github.com/user-attachments/assets/9d604851-8de5-47e7-9b80-3ab50c3f333f" />

**Comment by joewhitsitt** [comment]
Doubt we can control it in this Views/BEF situation, but alternatively we could be using fieldset/legend to group inputs. https://adrianroselli.com/2022/07/use-legend-and-fieldset.html

**Comment by pyrello** [comment]
I'm wondering if we could just remove the `role="group"` from these details elements. The `group` role is already the implicit ARIA role for this element type.

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/details#technical_summary

**Comment by bspeare** [comment]
Response from Siteimprove:

> Thank you for reaching out. Unfortunately, our support team isn't able to assist at a consultant level with Accessibility issues due to legal requirements about certifications for Web Content Accessibility Guidelines. We recommend checking the ALFA and WCAG documentation provided in the page report for the issue. It will show up on the left-hand side of the screen under the "Why is this important?" dropdown. 
> 
> If the content there doesn't answer your questions, we recommend submitting this as a false positive to have it reviewed by our Accessibility team. To do so, navigate to the page report for the issue, click the gear icon within the issue, and select "Dismiss as False Positive". You can also use a CSS selector to report multiple false positives at one time. You can learn more about this from our [Next Generation accessibility: How to dismiss occurrences of accessibility issues](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhelp.siteimprove.com%2Fsupport%2Fsolutions%2Farticles%2F80000962686-accessibility-how-to-dismiss-occurrences-of-accessibility-issues&data=05%7C02%7Cbenjamin-speare%40uiowa.edu%7Ca429231a64d2403482a908de2c6b7747%7C1bc445959aba4fc3b8ec7b94a5586fdc%7C1%7C0%7C638997037248502416%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=WBzs%2B97LDQkC3Lu1Z4qq9lsmFr4fMUjiNxObv7ER%2BjI%3D&reserved=0) help article. This will generate a report that is sent to our accessibility product team for review. Once they've reviewed it, you will receive an email notification to confirm the issue and award the points for the issue. 
> 
> Additionally, you can speak with your Customer Success Executive Erika Borgos ([ebor@siteimprove.com](mailto:ebor@siteimprove.com)) regarding what options you have available for professional services like Accessibility consultations.
> ​
> Thank you,
> --
> Travis N.
> Customer Support Engineer
> Siteimprove, Inc.
